money-core-scala/src/main/scala/com/comcast/money/core/akka/MoneyActor.scala

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2012-2015 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.comcast.money.core.akka
+
+import scala.collection.mutable.{ Buffer, Stack }
+import akka.actor.{ Actor, ActorSystem }
+import com.comcast.money.api.Span
+import com.comcast.money.core.{ DisabledSpan, Formatters, Tracer, Tracers }
+import com.comcast.money.core.internal.SpanLocal
+import spray.http.{ HttpHeader, Rendering }
```

Comment:
the big problem here is that you are coupling akka and spray

Spray is really no longer supported, akka http has made its way into akka 2.4.2, so isn't even a separate thing.

If you have to do something with Spray, have to create a separate module.  Moving forward, very rarely is anyone going to be using Spray